packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -78,15 +78,26 @@ export default function NewCommentModal({
     { enabled: input?.length > 0 },
   );
 
-  const confirmClose = (event: MouseEvent): void => {
+  const confirmClose = async (event: MouseEvent): Promise<void> => {
+    const options: PromptOptions = {
+      title: 'Discard comment',
+      description: 'Are you sure you want to close and discard your comment?',
+      okButton: {
+        className: 'btn-primary-ketchup',
+        title: 'Discard',
+      },
+      cancelButton: {
+        title: 'Stay',
+      },
+    };
```

Comment:
I think we can take this outside of the component and be declared just a const.

Reply:
Makes sense. Done.